src/App.js

```diff
@@ -12,16 +12,20 @@ import ForgotPassword from './components/ForgotPassword/ForgotPassword';
 import queryString from 'query-string';
 
 class App extends Component {
-  constructor({ locale, dependencies: { sessionManager } }) {
+  /**
+   * @param { Object } props - props
```

Comment:
it's common use the params here?

Reply:
I do not know, but it is the best and easy way that I found to have intellisense about input parameters:

![image](https://user-images.githubusercontent.com/1157864/55963708-b628b580-5c49-11e9-9925-f04a1ccb6925.png)